api/DHSC.FingertipsNext.Api/Program.cs

```diff
@@ -71,4 +75,35 @@ private static void RegisterModules(IServiceCollection services, ConfigurationMa
             module?.RegisterConfiguration(configuration);
         });
     }
+
```

Comment:
Can we move the context registration to the module please - each module should be standalone with no dependencies on the overall process or each other.  I'd suggest you put the database registration in the Module class and create a separate method called from RegisterModule to do this

Reply:
I've moved registration into `Module.cs`